tracer/src/Datadog.FleetInstaller/Commands/InstallCommand.cs

```diff
@@ -55,6 +56,37 @@ internal static ReturnCode Execute(
     {
         log.WriteInfo("Installing .NET tracer");
 
+        bool tryIisRollback;
+
+        try
+        {
+            log.WriteInfo("Checking IIS app pools for pre-existing instrumentation variable");
+            if (AppHostHelper.GetAppPoolEnvironmentVariable(log, Defaults.InstrumentationInstallTypeKey, out var value))
+            {
+                var expectedValue = Defaults.InstrumentationInstallTypeValue;
+                if (expectedValue.Equals(value, StringComparison.Ordinal))
+                {
+                    log.WriteInfo("Found existing instrumentation install type. Won't rollback IIS instrumentation if install fails");
+                    tryIisRollback = false;
+                }
+                else
+                {
+                    log.WriteInfo($"Found instrumentation install type, but did not have expected value {expectedValue}. Will rollback IIS instrumentation if install fails");
```

Comment:
Should we log the actual value or is that not important?

Reply:
Or is it because we don't _want_ to log the value as it may be sensitive?

Reply:
Updated in https://github.com/DataDog/dd-trace-dotnet/pull/6697/commits/0d0e103642080e2522fb8b23c6e77149eb649101